### ISSUE TITLE
FAR and reference highlighting doesn't work for variable ref inside tuple deconstructuion

### ISSUE THREAD

**Issue by genlu** [dotnet/roslyn#32982]
FAR and reference highlighting doesn't work for selected variable inside tuple deconstruction:

![image](https://user-images.githubusercontent.com/788783/52022438-bff5a600-24ad-11e9-98d3-a9fa0f6465e0.png)

Here's the result when selecting the same variable but outside the tuple deconstruction:

![image](https://user-images.githubusercontent.com/788783/52022494-f6332580-24ad-11e9-83bd-d80b0900aed9.png)

**Comment by genlu** [comment]
@jcouv is this a known issue?

**Comment by jcouv** [comment]
Yes, I think it's another symptom of https://github.com/dotnet/roslyn/issues/20115
Basically, if you point at `(x$$, y)`, you're pointing at `(int, int).Item1`, not `x`.

You can close this issue as duplicate and add this scenario to existing issue. Thanks

**Comment by CyrusNajmabadi** [comment]
Dupe of https://github.com/dotnet/roslyn/issues/20115
